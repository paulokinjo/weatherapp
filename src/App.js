import './App.css';

import WeatherInfoScreen from './screens/WeatherInfoScreen';

function App() {
  return (
    <div className="App">
      <WeatherInfoScreen />
    </div>
  );
}

export default App;
