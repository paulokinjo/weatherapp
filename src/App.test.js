import { render, screen } from '@testing-library/react';

import App from './App';
import { shallow } from 'enzyme';

describe('App', () => {
  const app = shallow(<App />);

  describe('Layout', () => {
    it('should render properly', () => {
      expect(app).toMatchSnapshot();
    });
  });
});
