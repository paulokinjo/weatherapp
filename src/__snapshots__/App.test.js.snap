// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Layout should render properly 1`] = `ShallowWrapper {}`;
